PLSS/src/app/run.js

```diff
@@ -1,6 +1,6 @@
 ((function () {
     var config = {
-        baseUrl: '/src',
+        baseUrl: '/.',
```

Comment:
**question**: this wasn't used in the version deployed correct?

Reply:
Yes, I believe that it was.